src/audio/pipeline/pipeline-schedule.c

```diff
@@ -181,6 +181,13 @@ void pipeline_schedule_triggered(struct pipeline_walk_context *ctx,
 			}
 			pipeline_schedule_copy(p, 0);
 		}
+		break;
+	case COMP_TRIGGER_XRUN:
+		list_for_item(tlist, &ctx->pipelines) {
+			p = container_of(tlist, struct pipeline, list);
+			if (!p->xrun_bytes)
+				p->xrun_bytes = 1;
```

Comment:
magic number

Reply:
any non-zero should do, the value isn't used anyway. I can add a comment.

Reply:
I see but can't we just assign exact XRUN_BYTES here? Even though currently we treat it as a boolean value at some point we may need exact value - this veriable was defined as 32 bit one so this suggest me that was the original intention.